.github/workflows/push.yml

```
@@ -7,23 +14,48 @@ on:
+            echo "TAG_CREATED=true" >> $GITHUB_ENV
+            echo "✅ Successfully created and pushed tag $VERSION"
+          else
+            echo "⏭️ Tag $VERSION already exists, skipping tag creation"
+            echo "TAG_CREATED=false" >> $GITHUB_ENV
+          fi
 
-      - name: Release Version 🚀
-        uses: opspresso/action-builder@master
+      - name: Create GitHub Release 🚀
+        if: env.TAG_CREATED == 'true' # Only create release if a new tag was created
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.4)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)